lib/storage/azure.js

```diff
@@ -183,54 +183,53 @@ module.exports = function() {
     },
 
     copyIn: function(localPath, _path, options, callback) {
+      const randStr = () => Math.random().toString(36).substring(7);
```

Comment:
Please use ordinary named functions for cases like this where they are worth naming, and place them after the end of the outer function (top-down programming) for readability, not at the top (this is one of the benefits of named functions).

Reply:
agreed

Reply:
this is extraneous - I thought this would be reused it is not.